### PR TITLE
0.2/757 login timeout error message

### PR DESCRIPTION
**Pull request by charlesstrange2**
Anyone know how to test the CSRF timeout?

### CODE REVIEW

**Comment by katjam** [comment]
I think you can set the expiry on the token to the past? Maybe.

File: config/session.php [hunk 1]
```diff
@@ -29,9 +29,9 @@
     |
     */
 
-    'lifetime' => 120,
+    'lifetime' => 1,
```
Comment (katjam) [inline config/session.php]: 1 what? Minute?

**Review by katjam — APPROVED** [review]
:leaves: Pretty swish!

File: config/session.php (hunk 1 above)
Comment (charlesstrange2) [reply config/session.php]: Ah. no. that would b bad. lets set it back to 60.

**Pull request merged**